src/components/OptionRow.js

```diff
@@ -70,6 +70,9 @@ const propTypes = {
     /** Whether to remove the lateral padding and align the content with the margins */
     shouldDisableRowInnerPadding: PropTypes.bool,
 
+    /** Whether to take focus when selecting */
+    shouldTakeFocus: PropTypes.bool,
```

Comment:
Can we make the name consistent? So use `shouldFocusOnSelectRow` and reverse conditions.

Reply:
@situchan 

When we should refocus on selecting row for options-selector, the option row should not take focus. I thought this makes sense.
Do you still want to rename the props?

Reply:
yeah, tricky naming. `shoudTakeFocus` is fine but no one knows what this does when read this code without any context.

Reply:
Maybe `shouldPreventDefaultFocusOnClick` though it's long name?

Reply:
I think it's fine

Reply:
Let's do that for now. Btw Monil might address this again during final review.

Reply:
@situchan 

Props name was changed.

Reply:
Yes, I found `shouldFocusOnSelectRow` and `shouldPreventDefaultFocusOnSelect` different and confusing if one doesn't know the context. I think we should use only one. Name and context wise both looks good.

Reply:
Do you agree to use `shouldFocusOnSelectRow` everywhere?

Reply:
If we still want to have the same props name, I suggest `shouldPreventDefaultFocusOnSelectRow`

Reply:
`shouldPreventDefaultFocusOnSelectRow` sounds good to me. 
I think also `shouldPreventDefaultFocusOnSelectItem` will work. 
But yes, using same prop through out the tree will be better IMO

Reply:
Okay. Let me rename the props for BaseOptionSelector, BaseOptionsList, BaseSelectionList, BaseListItem, and OptionRow

cc @situchan